joplin/base/models/janis_page.py

```diff
@@ -175,20 +167,34 @@ def janis_preview_url_end(self, revision=None):
             # Janis will query from its default CMS_API if a param is not provided
             return url_end + f"?CMS_API={settings.CMS_API}"
 
-    # Use hardcoded JANIS_URL for staging and prod
-    # Otherwise use configurable preview_janis_branch setting
-    def janis_url_base(self):
+    def janis_url_base(self, janis_branch):
+        """
+        returns a valid url of the base URL in janis:
+            Use hardcoded JANIS_URL for staging and prod
+            Otherwise, use configurable branch setting
+
+        TODO: this and url_base in site settings could probably
+        be revisited for semantics to be less confusing
+        """
         if settings.ISSTAGING or settings.ISPRODUCTION:
             return os.getenv("JANIS_URL")
         else:
-            return JanisBranchSettings.objects.first().branch_preview_url_base()
+            branch_settings = JanisBranchSettings.objects.first()
+            return branch_settings.url_base(janis_branch)
+
+    # alias for url base function
+    janis_preview_url_start = janis_url_base
```

Comment:
The `janis_preview_url_start` function can be gotten rid of. Just use janis_url_base now that we've got it.